lib/app/modules/uscore_v3.0.0/pediatric_bmi_for_age_sequence.rb

```diff
@@ -44,8 +42,7 @@ def validate_resource_item(resource, property, value)
 
       details %(
 
-        The #{title} Sequence tests `#{title.gsub(/\s+/, '')}` resources associated with the provided patient.  The resources
-        returned will be checked for consistency against the [PediatricBmiForAge Argonaut Profile](http://hl7.org/fhir/us/core/StructureDefinition/pediatric-bmi-for-age)
+        The #{title} Sequence tests `#{title.gsub(/\s+/, '')}` resources associated with the provided patient.
```

Comment:
Was this change intended?

Reply:
Yes.  This broke... something.  I'm not exactly sure what.  But this whole section should be a lot more specific.

Reply:
I'm going to leave this as-is.  I think the whole details section needs to be a lot more descriptive anyhow (it should pull content out of the `description` field in the profiles, describe the test approach, etc).  I added a ticket for more robust documentation being generated here, see FI-375